lib/promoted/ruby/client/defaults.rb

```diff
@@ -0,0 +1,17 @@
+module Promoted
+  module Ruby
+    module Client
+      USE_CASES = {'UNKNOWN_USE_CASE'=> 0,
```

Comment:
why aren't use using the ruby generated protobuf enum?

Reply:
Enums are majorly for models in rails and they add extra functionality that help query database. We don't need that functionality here. So it's better to keep it simple.